### PR TITLE
DeveloperGuide.adoc: Setting Up updates

### PR DESCRIPTION
**Pull request by afiqlattif**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -44,23 +44,26 @@ To get started, head on to <<Setting-Up>>.
...
-. *IntelliJ* IDE
+* *IntelliJ* IDE
 +
 [NOTE]
 IntelliJ by default has Gradle and JavaFx plugins installed. +
 Do not disable them. If you have disabled them, go to `File` > `Settings` > `Plugins` to re-enable them.
 
 
 === Setting up the project in your computer
+Here are the steps to setup the project in your computer:
```
Comment (Kratious) [inline docs/DeveloperGuide.adoc]: I think it's set up instead of setup. Setup is a noun

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -44,23 +44,26 @@ To get started, head on to <<Setting-Up>>.
...
 
 . Fork this repo, and clone the fork to your computer
```
Comment (iMarbles) [inline docs/DeveloperGuide.adoc]: Maybe it would be better to give the link to the repo?

Comment (iMarbles) [inline docs/DeveloperGuide.adoc]: Missing full stop for every instruction

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -74,39 +77,46 @@ Do not disable them. If you have disabled them, go to `File` > `Settings` > `Plu
...
+To resolve this, place your cursor over any of the code section highlighted in red.
+Press kbd:[ALT + ENTER], and select `Add '--add-modules=...' to module compiler options` for each error
 . Repeat this for the test folder as well (e.g. check link:{repoURL}/src/test/java/seedu/address/commons/util/XmlUtilTest.java[`XmlUtilTest.java`] and link:{repoURL}/src/test/java/seedu/address/ui/HelpWindowTest.java[`HelpWindowTest.java`] for code errors, and if so, resolve it the same way)
 
 === Verifying the setup
+To verify that you have successfully set up your project on your computer, you must:
 
-. Run the `seedu.address.MainApp` and try a few commands
-. <<Testing,Run the tests>> to ensure they all pass.
+* Run the `seedu.address.MainApp` and try a few commands
```
Comment (iMarbles) [inline docs/DeveloperGuide.adoc]: Missing full stop after for 'Run the.....'

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -74,39 +77,46 @@ Do not disable them. If you have disabled them, go to `File` > `Settings` > `Plu
...
-* For `Class count to use import with '\*'` and `Names count to use static import with '*'`: Set to `999` to prevent IntelliJ from contracting the import statements
-* For `Import Layout`: The order is `import static all other imports`, `import java.\*`, `import javax.*`, `import org.\*`, `import com.*`, `import all other imports`. Add a `<blank line>` between each `import`
+. Set `Class count to use import with '\*'` and `Names count to use static import with '*'` to `999` to prevent IntelliJ from contracting the import statements
+. Set the order of `Import Layout` to `import static all other imports`, `import java.\*`, `import javax.*`, `import org.\*`, `import com.*`, `import all other imports` and add a `<blank line>` between each `import`
 
++
+[NOTE]
 Optionally, you can follow the <<UsingCheckstyle#, UsingCheckstyle.adoc>> document to configure Intellij to check style-compliance as you write code.
 
 ==== Updating documentation to match your fork
```
Comment (Kratious) [inline docs/DeveloperGuide.adoc]: Do we still need this section?
Since our repo already removes the SE-EDU branding, this section seems out of place.

Comment (Kratious) [inline docs/DeveloperGuide.adoc]: Do we still need this section?
Since our repo already removes the SE-EDU branding, this section seems out of place.

**Comment by Kratious** [comment]
Looks good!

**Pull request merged**